crates/holochain_zome_types/Cargo.toml

```diff
@@ -70,7 +69,7 @@ default = ["full-dna-def"]
 # but used in Holochain
 full-dna-def = [
   "derive_builder",
-  "nanoid",
+  "dep:uuid",
```

Comment:
`nanoid` also uses an old version of `rand` and we really don't need ids in a particular format here. `uuid` also needs `rand` but it's more actively maintained and has already released with support for the new `rand`